recipe/meta.yaml

```diff
@@ -23,9 +23,7 @@ requirements:
     - pip
   run:
     - python
-    - numpy >=1.8  # [not (win and (py35 or py36 or py37))]
-    - numpy >=1.9  # [win and py35]
-    - numpy >=1.11  # [win and py36 or py37]
```

Comment:
@isuruf, @peterjc can you comment on why these specifications were/ are not necessary anymore?

Reply:
This dates back to efdc03832a8b04a3b1945ade4d3fd244337bc4c5 by @kain88-de 

My guess is this was driven by something about the state of the available NumPy packages at the time. For example, with https://github.com/biopython/biopython-wheels/ we set a minimum NumPy according to the version of Python